pkg/crc/api/api.go

```diff
@@ -44,18 +31,35 @@ func CreateApiServer(socketPath string) (CrcApiServer, error) {
 }
 
 func (api CrcApiServer) Serve() {
+	go api.handleClusterOperations() // go routine that handles start, stop and delete calls
 	for {
 		conn, err := api.listener.Accept()
 		if err != nil {
 			logging.Error("Error establishing communication: ", err.Error())
 			continue
 		}
-		go api.handleConnection(conn)
+		api.handleConnections(conn) // handle version, status, webconsole, etc. requests
```

Comment:
what is the motivation behind removing the goroutine?

Reply:
Now the go routine is called from `api.handleConnections` to make the cluster operations `start/stop/delete` be served in a different go routine.